.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the code
+      uses: actions/checkout@v3
+
+    - name: Install Poetry
+      run: |
+        curl -sSL https://install.python-poetry.org | python3 -
+      shell: bash
+
+    - name: Install dependencies
+      run: poetry install
```

Comment:
you may need poetry install --with test to install all dependencies (see README)